src/compliment/sources/namespaces_and_classes.clj

```diff
@@ -94,11 +95,12 @@
              ;; If prefix doesn't contain a period, using fuziness produces too many
              ;; irrelevant candidates.
              (for [{:keys [^String ns-str, ^String file]} (utils/namespaces&files-on-classpath)
+                   :let [[_ quoted prefix] (re-matches #"(@|@?#'|')?(.+)" prefix)]
```

Comment:
Three usages of a not-so-trivial regex, I think this can be extracted into a function.